appdatabase/migrations/sql/0001_app.up.sql

```diff
@@ -56,8 +56,8 @@ sender VARCHAR NOT NULL,
 receipt BLOB,
 log BLOB,
 type VARCHAR NOT NULL,
-FOREIGN KEY(network_id,blk_hash) REFERENCES blocks(network_id,hash) ON DELETE CASCADE,
```

Comment:
Why were these removed?

Reply:
because I used a different table for this PR, as I didn't want to touch code which is not related to scanning. I can rename `blocks_with_transfers` to `blocks` now and put this key back. 

The difference is that `blocks` was used for saving blocks while app is running (besides scanning), and we stored even blocks which did not contain transaction we want to fetch, and it feels like unnecessary thing which might also create a lot of garbage in local db.